sdk/metricsadvisor/Azure.AI.MetricsAdvisor/README.md

```diff
@@ -1,45 +1,466 @@
+
+Headers:
+X-Request-ID: REDACTED
+x-envoy-upstream-service-time: REDACTED
+apim-request-id: REDACTED
+Strict-Transport-Security: REDACTED
+X-Content-Type-Options: REDACTED
+Date: Thu, 08 Oct 2020 09:04:31 GMT
+Content-Length: 69
+Content-Type: application/json; charset=utf-8
+```
+
+### Setting up console logging
+
+The simplest way to see the logs is to enable console logging.
+
+To create an Azure SDK log listener that outputs messages to the console use the `AzureEventSourceListener.CreateConsoleLogger` method.
+
+```C#
+// Set up a listener to monitor logged events.
+using AzureEventSourceListener listener = AzureEventSourceListener.CreateConsoleLogger();
+```
+
+To learn more about other logging mechanisms see [Diagnostics Samples][logging].
 
 ## Next steps
 
-Forthcoming
+Samples showing how to use the Cognitive Services Metrics Advisor library are available in this GitHub repository. Samples are provided for each main functional area:
+
+- [Create a data feed from a data source][metricsadv_sample1]
+- [Check the ingestion status of a data feed][metricsadv_sample2]
+- [Create an anomaly detection configuration][metricsadv_sample3]
+- [Create a hook for receiving anomaly alerts][metricsadv_sample4]
+- [Create an anomaly alert configuration][metricsadv_sample5]
+- [Query detected anomalies and triggered alerts][metricsadv_sample6]
 
 ## Contributing
 
-This is a template, but your SDK readme should include details on how to contribute code to the repo/package.
+This project welcomes contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit [cla.microsoft.com][cla].
 
-<!-- LINKS -->
-[style-guide-msft]: https://docs.microsoft.com/style-guide/capitalization
-[style-guide-cloud]: https://aka.ms/azsdk/cloud-style-guide
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct][code_of_conduct]. For more information see the [Code of Conduct FAQ][coc_faq] or contact [opencode@microsoft.com][coc_contact] with any additional questions or comments.
 
 ![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-net%2Fsdk%2Fmetricsadvisor%2FAzure.AI.MetricsAdvisor%2FREADME.png)
+
+<!-- LINKS -->
+[metricsadv_client_src]: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src
+<!--[metricsadv_docs]: https://docs.microsoft.com/en-us/azure/cognitive-services/metrics-advisor
+[metricsadv_nuget_package]: https://www.nuget.org/packages/Azure.AI.MetricsAdvisor-->
```

Comment:
We can use this as a placeholder until the package releases:
`https://www.nuget.org/packages?q=Azure.AI.MetricsAdvisor`